quarkus/dist/src/main/content/providers/README.md

```diff
@@ -3,8 +3,8 @@ Installing Custom Providers
 
 Add your custom provider JAR files in this directory.
 
-Once you have your providers in this directory, run the following command to complete the installation:
+Once you have your providers files in this directory, run the following command to complete the installation:
```

Comment:
```suggestion
Once you have your providers in this directory, run the following command to complete the installation:
```

providers files is not correct, would be provider files. However, think just `providers` is better?